src/vm/i386/unixstubs.cpp

```diff
@@ -6,7 +6,11 @@
 
 extern "C"
 {
-    void ThrowControlForThread()
+    void ThrowControlForThread(
+#ifdef WIN64EXCEPTIONS        
```

Comment:
This ifdef can go away with this change that switches to WIN64EXCEPTIONS on x86 Linux.